### PR TITLE
[CAL-1125] Fix app detail page heading size

### LINKED ISSUE #7246: [CAL-1125] back arrow not centered in app detail page
Screenshot 2023-02-21 at 11-40-25 Zoom Video Cal com

<sub>[CAL-1125](https://linear.app/calcom/issue/CAL-1125/back-arrow-not-centered-in-app-detail-page)</sub>

### PR DESCRIPTION
**Pull request by leventdeniz**
## What does this PR do?

Fixes #7246

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by CarinaWolli — DISMISSED** [review]
LGTM! Thank you 🙏

**Review by esau-morais — COMMENTED** [review]
I don't think that's enough to adapt to all pages, once `Shell` is reused in many of them. If you analyse the back link, it has a `mt-2`, which will not get to center in some other cases.

Adding up to your solution, I'd probably do something like this in the `packages/features/shell/Shell.tsx` file

image

**Comment by sean-brydon** [comment]
> I don't think that's enough to adapt to all pages, once `Shell` is reused in many of them. If you analyse the back link, it has a `mt-2`, which will not get to center in some other cases.
> 
> Adding up to your solution, I'd probably do something like this in the `packages/features/shell/Shell.tsx` file
> 
> image

Stole exactly what i was going to say :D 

Fellow lazygit user o/

**Comment by leventdeniz** [comment]
dont know how I didnt come up with that, thanks.
Just fyi, this centers other elements too but I guess thats what we want

https://user-images.githubusercontent.com/32421538/220634161-23063fdc-7e05-44f2-b8fc-9e8ef8de6b0f.mov

**Review by esau-morais — APPROVED** [review]
LGTM

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -780,7 +780,11 @@ export function ShellMain(props: LayoutProps) {
 
   return (
     <>
-      <div className={classNames("mb-4 flex sm:mt-0 md:mb-6", props.smallHeading ? "lg:mb-7" : "lg:mb-8")}>
+      <div
+        className={classNames(
+          "mb-4 flex items-center sm:mt-0 md:mb-6",
+          props.smallHeading ? "lg:mb-7" : "lg:mb-8"
+        )}>
```
Comment (esau-morais) [inline packages/features/shell/Shell.tsx:787]: I'd recommend you to commit a simple change, which will also solve #7290 
```suggestion
        className={classNames(
          "my-4 flex items-center md:mt-0 md:mb-6",
          props.smallHeading ? "lg:mb-7" : "lg:mb-8"
        )}>
```

**Review by esau-morais — CHANGES_REQUESTED** [review]

File: packages/features/shell/Shell.tsx (hunk 1 above)
Comment (leventdeniz) [reply packages/features/shell/Shell.tsx:787]: Pushed it, thanks!

**Pull request merged**